pages/docs/dragonite/config.mdx

```diff
@@ -89,20 +106,41 @@ bearer_token = "KOJI_SECRET"
 [tuning]
 #recycle_gmo_limit = 4900
 #recycle_encounter_limit = 9900
+#recycle_invasion_limit = 940
 #recycle_on_jail=false
 #minimum_account_reuse_hours = 169
-#location_delay = 0
-#fort_location_delay = 0
+#scout_age_limit = 30
+location_delay = 3000
+fort_location_delay = 3000
+quest_layer_swap = false
+test_shadow_banned = true
+
 ```
 
 | Key | Default | Description |
 | :-- | :-- | :-- |
 | `recycle_gmo_limit` | 4900 | (Advanced) Recommend keeping this value commented out. When the GMO limit is reached for a specific account it will be be rotated. |
 | `recycle_encounter_limit` | 9900 | (Advanced) Recommend keeping this value commented out. When an account has encountered this amount of pokemon it will be rotated. |
+| `recycle_invasion_limit` | 940 | (Advanced) Recommend keeping this value commented out. When an account has encountered this amount of invasions it will be rotated. |
 | `recycle_on_jail` | false | Enabling this setting will utilize significantly more accounts. Currently five accounts are assigned to one worker and will rotate while the worker is running. |
-| `location_delay` | 0 | This value, in milliseconds, will add a slight wait period while a worker is moving to a new location. Adding a slight delay has proven to decrease empty GMO results and positively impact scans overall. Recommend starting with 100, 200, 300, etc and monitoring overall results. |
+| `location_delay` | 0 | This value, in milliseconds, will add a slight wait period while a worker is moving to a new location. Adding a slight delay has proven to decrease empty GMO results and positively impact scans overall. Recommend starting with 2500+ and monitoring overall results. |
 | `fort_location_delay` | 0 | This value, in milliseconds, is the same as `location_delay` however it only impacts fort scans (gyms/pokestops). |
 | `minimum_account_reuse_hours` | 0 | This sets the amount of hours for accounts to "rest" until they can be reused. Recommended value is 169 hours (7 days). |
+| `scout_age_limit` | 30 | Configure age limit for scout queue entries |
+| `test_shadow_banned` | false | Automatically test accounts for being shadow banned and marked as banned in database. |
+| `quest_layer_swap` | true | This option for questing will hold on to AR Quests and try to move to 'ar only scans' over time. This may become default later (the current default is 'true' which means it will try and drop ar quests if you have them as required, unless your account is marked as ar banned). |
+
+### Dynamic Location Delay
+Make `location_delay`(max) dynamic by tuning against `initial_gmo_success_rate`(percentage), limited by `minimum_location_delay`(min).
+
+This is applied per worker and resets to the max value(location_delay) after an account is rotated. Therefore this should NOT be used with `recycle_on_jail` set to true.
```

Comment:
the note with `recycle_on_jail` might then be obsolete as well, as config gets removed :)